app/lib/actions/katello/repository/sync.rb

```diff
@@ -44,11 +45,11 @@ def plan(repo, pulp_sync_task_id = nil, options = {})
               plan_action(Pulp::Repository::Download, :pulp_id => repo.pulp_id, :options => {:verify_all_units => true}) if validate_contents
               plan_action(Katello::Repository::MetadataGenerate, repo, :force => true) if skip_metadata_check
```

Comment:
Shouldnt you skip this too for non yum?

Reply:
skip_metadata_check is a very specific option that is passed in, i'm not 100% sure what you mean?